rust/src/ike/ikev1.rs

```diff
@@ -40,27 +40,27 @@ pub struct IkeV1Header {
 pub struct Ikev1ParticipantData {
     pub key_exchange: String,
     pub nonce: String,
-    pub vendor_ids: HashSet<String>,
-    /// nested Vec, outer Vec per Proposal/Transform, inner Vec has the list of attributes.
-    pub transforms: Vec<Vec<SaAttribute>>,
+    pub nb_transforms: usize,
```

Comment:
could this be explicit type? usize depends on the architecture. Traffic properties should not.

Reply:
Ok, I wanted to avoid the rust casts